routers/web/repo/issue.go

```diff
@@ -419,9 +419,6 @@ func RetrieveRepoMilestonesAndAssignees(ctx *context.Context, repo *models.Repos
 	}
 
 	handleTeamMentions(ctx)
-	if ctx.Written() {
```

Comment:
This seems unwise

Reply:
yes

Reply:
ah I missed that this is the end of the function so it's fine!